app/lib/sprites/SpriteExporter.coffee

```diff
@@ -24,10 +24,14 @@ class SpriteExporter extends CocoClass
 
   build: ->
     spriteSheetBuilder = new createjs.SpriteSheetBuilder()
+
     if @spriteType is 'segmented'
       @renderSegmentedThangType(spriteSheetBuilder)
+    else if @spriteType is 'animate'
+      throw new Error('Animate sprite type not currently supported in sprite exporter')
```

Comment:
Searched for all occurrences of spriteType in the client side code base and added explicit handling of animate sprite type.